tests/validation_test.go

```diff
@@ -24,9 +24,9 @@ func TestMaxStringLength(t *testing.T) {
 			wantErr: nil,
 		},
 		{
```

Comment:
I'll probably add another test case to check that a value bigger than `maxLenght`returns error as expected

Reply:
Thanks for the suggestion. I've added a test for that 👍